src/Authoring/WinRT.SourceGenerator/Generator.cs

```diff
@@ -295,7 +298,18 @@ public static int DllGetActivationFactory(void* activatableClassId, void** facto
             }
             else if (ShouldEmitCallToTryGetDependentActivationFactory(context))
             {
-                builder.AppendLine("IntPtr obj = TryGetDependentActivationFactory(fullyQualifiedTypeName);");
+                builder.AppendLine("""
+                                   IntPtr obj = TryGetDependentActivationFactory(fullyQualifiedTypeName);
+                    """);
+                builder.AppendLine();
+            }
+            else
+            {

```

Comment:
Good catch with this, I was actually debugging a scenario earlier today that was resulting in a compiler error here.  Still need to debug separately why it wasn't seeing a winrt component.